app/controllers/api/v1/api_users_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::V1::ApiUsersController < Api::V1::ApiController
+  before_action :validate_create_params, only: %w[create]
+  before_action :validate_show_params, only: %w[show]
+
+  def create
+    api_user = create_api_user(name: params.fetch(:name), email: params.fetch(:email))
+
```

Comment:
tiny whitespace nit (just so it's consistent with the one below)
```suggestion
```

Reply:
Fixed :)